### DISCUSSION TITLE
How to connect with api using js

### DISCUSSION THREAD

**Discussion by saibabu10** [lllyasviel/Fooocus#2691] [Q&A] [answered]
i have tried a lot i dont know how to connect with the foooocus which is running in google collab so please kindly guide me and i am using react to get the data from the https://f9d3b6044fab6916a4.gradio.live

**Comment by mashb1t** [comment, marked as answer]
https://github.com/lllyasviel/Fooocus/discussions/2613#discussioncomment-8973861
